code/modules/mob/living/carbon/human/life.dm

```diff
@@ -610,6 +610,19 @@
 			if(prob(nausea))
 				delayed_vomit()
 
+		if(CE_ITCH in chem_effects)
+			var/itching = chem_effects[CE_NOITCH]
+			if(CE_ITCH in chem_effects)
+				itching -= chem_effects[CE_NOITCH]
+			if(itching > 1)
+				if(prob(5))
+					to_chat(src, SPAN_NOTICE(pick("You have an annoying itch.", "You have a slight itch.")))
+			if(itching > 3)
+				if(prob(10))
+					to_chat(src, SPAN_WARNING(pick("The itch is becoming progressively worse.", "You need to scratch that itch!", "The itch isn't going!")))
+					src.emote("scratch")
+					src.take_organ_damage(4, 0)
```

Comment:
That's a pretty decent amount of organ damage. How bad does this get, usually?

Reply:
Wasn't too bad due to the low chance of it occurring. I've scrapped the emote/brute damage however, flavour messages are fine on their own, imo.